docker-compose.yml

```diff
@@ -1,8 +1,7 @@
-version: "3.7"
```

Comment:
Plus besoin de numéro de version ? Ça prend la dernière par défaut ? Tu peux préciser un peu à quoi servent les changements dans ce fichier ?

Reply:
Yes, je pensais être le seul utilsateur restant sur docker donc je n'ai pas détaillé 😅
- renommage en `docker-compose.yml`: maintenant qu'on n'a plus de compose de prod, autant que le seul fichier compose ait le nom "attendu". C'est plus facile à lancer
- la version est en effet optionelle depuis quelque temps (cf https://docs.docker.com/compose/ `optional since v1.27.0`)
- j'ai mis un `DOCKER_RESTART_POLICY` avec valeur par défault. Ca permet si on le veut de ne pas restart les containers automatiquement au lancement de Docker (si tu as une env `DOCKER_RESTART_POLICY=no). Si pas d'env c'est la valeur par défaut `unless-stopped` qui est utilisée. J'aurais pu laisser `always` en valeur par défaut mais ce n'est pas très différent...
- il y avait des variables d'env déjà définies dans le fichier `.env`. Pas besoin de les déclarer en doublon à priori, on a plutot tendance à mettre à jour ce fichier d'environnement

Reply:
Non t'es pas le seul :)